src/components/menu-top/LoadModal.vue

```diff
@@ -6,7 +6,7 @@
 
     <section class="modal-card-body">
       <b-field class="file">
-        <b-upload v-model="file" accept=".shw" @input="loadShow">
+        <b-upload v-model="file" @input="loadShow">
```

Comment:
Is there any way to specify both `.shw` and `.shw4`?

Reply:
Yes! I just needed a comma haha